src/Worker.cs

```diff
@@ -31,10 +31,34 @@ public async static Task Main(string[] args)
                 LogLevel.Information,
                 string.Format(PowerShellWorkerStrings.PowerShellWorkerVersion, typeof(Worker).Assembly.GetName().Version));
 
-            WorkerArguments arguments = null;
-            Parser.Default.ParseArguments<WorkerArguments>(args)
-                .WithParsed(ops => arguments = ops)
-                .WithNotParsed(err => Environment.Exit(1));
+            var workerOptions = new WorkerOptions();
+
+            var parser = new Parser(settings => settings.EnableDashDash = true);
+            parser.ParseArguments<WorkerArguments>(args)
+                .WithParsed(workerArgs =>
+                {
+                    workerOptions.WorkerId = workerArgs.FunctionsWorkerId ?? workerArgs.WorkerId;
+                    workerOptions.RequestId = workerArgs.FunctionsRequestId ?? workerArgs.RequestId;
```

Comment:
This does constitute a behavior change, whereas previously the worker would exit if the WorkerId, RequestId, Host, or Port were null, now it will continue to run (and likely crash later). Not sure if that was intended

Reply:
The intention was for the parser to be less strict (not to fail if the command line arguments contain unknown parameters). However, I do agree that we should validate the WorkerOptions before continuing execution. I have addressed that.